interface/reports/criteria.tab.php

```diff
@@ -172,7 +176,7 @@
                     <div class= "col-xs-12">
                         <label class="control-label" for=""><?php echo text($ThisPageSearchCriteriaDisplay[$ThisPageSearchCriteriaIndex]); ?>:</label>
                         <div <?php //Don't Use class =  'form-control'?>>
-                            <?php $FunctionName=$ThisPageSearchCriteriaIncludeMaster[$ThisPageSearchCriteriaInclude[$ThisPageSearchCriteriaIndex]];
+                            <?php $FunctionName = "OpenEMR\\Billing\\" . $ThisPageSearchCriteriaIncludeMaster[$ThisPageSearchCriteriaInclude[$ThisPageSearchCriteriaIndex]];
```

Comment:
Would revert above line change and make the recommended change in the above interface/billing/billing_report.php script